### PR TITLE
Fix docs of this repo and add master branch protection

### PR DESCRIPTION
**Pull request by wu-sheng**
I noticed the helm chart doc is hidden is a very small link. It is easy to miss in the first read. I think we should merge them.

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -31,3 +31,8 @@ github:
     squash:  true
     merge:   false
     rebase:  false
+  protected_branches:
+    master:
+      required_pull_request_reviews:
+        dismiss_stale_reviews: true
+        required_approving_review_count: 1
```
Comment (wu-sheng) [inline .asf.yaml:38]: Add review requirement and branch protection.

File: .asf.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@
 #
 
 github:
-  description: Apache SkyWalking Kubernetes Deployment Helm Chart
+  description: BanyanDB Helm Chart repository provides ways to install and configure BanyanDB running in a cluster natively on Kubernetes.
```
Comment (wu-sheng) [inline .asf.yaml:19]: New repository description.

File: README.md [hunk 3]
```diff
@@ -7,17 +7,119 @@ Apache SkyWalking BanyanDB Helm Chart
 BanyanDB, as an observability database, aims to ingest, analyze and store Metrics, Tracing, and Logging data.
 It's designed to handle observability data generated by observability platforms and APM systems, like [Apache SkyWalking](https://github.com/apache/skywalking) etc.
 
-BanyanDB Helm Chart repository provides ways to install and configure SkyWalking in a BanyanDB cluster. The scripts are written in Helm 3.
+BanyanDB Helm Chart repository provides ways to install and configure BanyanDB running in a cluster natively on Kubernetes. The scripts are written in Helm 3.
```
Comment (wu-sheng) [inline README.md:10]: @hanahmily Please confirm this, I think this helm chart doesn't control skywalking OAP in any case.

**Comment by wu-sheng** [comment]
BTW @hanahmily @ButterBright Would you add a CI test in the repository to verify the real installation?

**Comment by ButterBright** [comment]
Sure.

**Pull request merged**